src/styles/calendar.scss

```diff
@@ -0,0 +1,45 @@
+.calendar {
+  display: flex;
+  flex-wrap: wrap;
+  gap: $cellsGap;
+  padding: $calendarPadding;
+  width: $calendarWidth;
+
+  &__day {
+    display: flex;
+    justify-content: center;
+    align-items: center;
+
+    width: $sizeOfSell;
+    height: $sizeOfSell;
+    border: 1px solid #404040;
+    background-color: #eee;
+
+    transition: all 0.5s;
+
+    &:hover {
+      cursor: pointer;
+      background-color: #ffcdd5;
+      transform: translateY(-20px);
+    }
+
+    @for $i from 1 through $monthLength {
+      &:nth-child(#{$i})::before {
+        content: "#{$i}";
+        font-size: 30px;
```

Comment:
You don't need to specify font size for each `nth-child` as it's the same, you can just specify to `::before` selector